README-NE.md

```diff
@@ -137,4 +137,4 @@ linksहरू खोज्नुहोस् जुन GitHub मा योग
 - [24 Pull Requests](https://24pullrequests.com) - चौबीस(24) Pull Request  डिसेम्बर महिनाको अवधिमा Open Source सहयोगलाई बढावा दिने प्रोजेक्ट हो।
 
 ## लाइसेन्स
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />यो कार्य <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a> अन्तर्गत लाइसेन्स प्राप्त छ।
```

Comment:
This can be kept and added back in.

Reply:
done e8c131ea1774d90dd3a30cabd529c5f16d1ce230